src/test_fixture/world.rs

```diff
@@ -237,6 +237,7 @@ where
         <[_; 3]>::try_from(output).unwrap()
     }
 
+    #[allow(unused_must_use)]
```

Comment:
Please change the test to use the output if that is what you need to do.

Reply:
Realised this validate does not need to be async anymore.